test/fops/fops.c

```diff
@@ -123,15 +98,17 @@ fop(void *arg)
 {
 	STATS *s;
 	uintptr_t id;
+	uint32_t rnd[2];
 	u_int i;
 
 	id = (uintptr_t)arg;
 	sched_yield();		/* Get all the threads created. */
 
 	s = &run_stats[id];
+	__wt_random_init(rnd);
 
 	for (i = 0; i < nops; ++i, sched_yield())
-		switch (r() % 9) {
+		switch (__wt_random(rnd) % 9) {
```

Comment:
I don't think it makes much difference -- if there was a reason to move `mmrand` into the test utils file, sure, otherwise, meh.